canvas_modules/common-canvas/src/toolbar/toolbar.jsx

```diff
@@ -301,7 +299,8 @@ Toolbar.propTypes = {
 	instanceId: PropTypes.number,
 	toolbarActionHandler: PropTypes.func,
 	tooltipDirection: PropTypes.string,
-	additionalText: PropTypes.object
+	additionalText: PropTypes.object,
+	size: PropTypes.string
```

Comment:
PropTypes can be declared to have an enumeration of values. So in this could use that to say specifically that only "sm" and "md" are allowed rather than this which only says size should be a string.

Reply:
Adjusted only for values : 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/89609176/181652689-33884a38-ad23-412f-9966-5a497c73ff2e.png">